docs/dev_101_states.md

```diff
@@ -37,6 +37,21 @@ def setup(hass, config):
     return True
 ```
 
+You'll also need to create the file `<config dir>/custom_components/hello_state/manifest.json` and copy the below example code.
+
+```python
+{
+    "domain": "hello_state",
+    "name": "Hello State",
+    "version": "0.1",
+    "documentation": "https://www.example.com",
+    "dependencies": [],
+    "codeowners": [],
+    "requirements": [],
```

Comment:
We should not showcase blank values

Reply:
That is fair.  As I am a complete noob, I copied and pasted the minimal example code from here: https://developers.home-assistant.io/docs/creating_integration_manifest/

I can add in dummy data; should I submit a PR for the other page also?

Reply:
Further thinking, it seems like I can only add a codeowner, as there are no dependencies or requirements for this hello world-type component.  Do you agree?

Reply:
I'd say keep this one as simple as possible, and link to the manifest docs for more details

Reply:
Perfect, will do!  Thanks for the feedback!

Reply:
Hey @peddamat! Small and friendly ping :)